resultcapture-service/src/main/kotlin/com/devotion/capture/service/serviceInterface.kt

```diff
@@ -0,0 +1,19 @@
+package com.devotion.capture.service
```

Comment:
somehow I don't like the name of this file `serviceInterface.kt`. I'd just call it `services.kt` or similar